node_modules/oae-authz/tests/test-groups.js

```diff
@@ -15,10 +15,12 @@
 
 var _ = require('underscore');
 var assert = require('assert');
+var shortid = require('shortid');
```

Comment:
I'm not sure. I think we started using PascalCasing all variables requiring modules of our own (e.g.,  PrincipalsDAO, ContentAPI, ..) but kept short single word variables such as `fs`, `assert`, `util`, ... lowercase? Either way it's all terribly inconsistent. I'd rather we keep using `ShortId` everywhere for now or settle on style and start changing all of them.